packages/logs/src/rawLogsEvent.types.ts

```diff
@@ -37,9 +31,11 @@ export interface RawLoggerLogsEvent extends CommonRawLogsEvent {
   origin: typeof ErrorSource.LOGGER
 }
 
-export interface RawNetworkLogsEvent extends CommonRawErrorLogsEvent {
+export interface RawNetworkLogsEvent extends CommonRawLogsEvent {
   date: TimeStamp
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏date should not be needed here anymore